ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -1089,9 +1089,8 @@ private boolean needBootstrapAcidTablesDuringIncrementalDump() {
     // of the ACID tables might be included for bootstrap during incremental dump. For old policy, its because the table
     // may not satisfying the old policy but satisfying the new policy. For filter, it may happen that the table
     // is renamed and started satisfying the policy.
-    return ((!work.replScope.includeAllTables())
-            || (previousReplScopeModified())
-            || conf.getBoolVar(HiveConf.ConfVars.REPL_BOOTSTRAP_ACID_TABLES));
+    return !work.replScope.includeAllTables() || previousReplScopeModified() || !tablesForBootstrap.isEmpty()
```

Comment:
do we need to update the comment above?

Reply:
Actually, function name "needBootstrapAcidTablesDuringIncrementalDump" is itself intuitive enough to signify that if tablesForBootstrap is non-empty, we need to bootstrap them.